src/tool/subcommands/benchmark_cmd.rs

```diff
@@ -112,6 +128,36 @@ async fn benchmark_forest_encoding(
     Ok(())
 }
 
+// Exporting combines a graph traversal with ForestCAR.zst encoding. Ideally, it
+// should be no lower than `min(benchmark_graph_traversal,
+// benchmark_forest_encoding)`.
+async fn benchmark_exporting(
+    input: Vec<PathBuf>,
+    compression_level: u16,
+    frame_size: usize,
+) -> Result<()> {
+    let store = open_store(input)?;
+    let heaviest = store.heaviest_tipset()?;
+    let stateroot_lookup_limit = heaviest.epoch() - 2000;
```

Comment:
It might be worth commenting on where the 2000 value comes from.

Reply:
Oh yeah, this should be a command-line parameter.